lib/SimpleSchema.js

```diff
@@ -591,6 +591,32 @@ class SimpleSchema {
     return def.label;
   }
 
+  /**
+   * Gets a field's property
+   *
+   * @param {String} [key] The schema key, specific or generic.
+   *   Omit this argument to get a dictionary of all labels.
+   * @param {String} [prop] Name of the property to get.
+   *
+   * @returns {any} The property value
+   */
+  get(key, prop) {
+    const def = this.getDefinition(key, ['type', prop]);
+
+    if (!def) return undefined;
+
+    if (schemaDefinitionOptions.includes(prop)) {
```

Comment:
@dpankros i think you might misunderstood each other. My intention was not to add another dependency(babel-polyfill), just wanted you to add this to be more certain about the reason of this error.

> is there something that Array.prototype.includes does that _.includes does not?
Actually, the reason why i used native Array function over the one from the lodash package is that probably in the future we might limit dependencies and get rid of lodash package.

If @aldeed prefers lodash functions over native JavaScript functions, i can change this.